### PR TITLE
GUVNOR-2600: Expose Dynamic Bean Validation through DataModelOracle

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-2600

### CODE REVIEW

**Comment by manstis** [comment]
Part of an ensemble:

https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/pull/234
https://github.com/droolsjbpm/drools/pull/828
https://github.com/droolsjbpm/kie-wb-common/pull/402
https://github.com/droolsjbpm/drools-wb/pull/236

File: drools-wb-webapp/pom.xml [hunk 1]
```diff
@@ -858,6 +858,24 @@
       <scope>provided</scope>
     </dependency>
 
+    <!-- GWT/Errai Bean Validation - JSR303 -->
```
Comment (manstis) [inline drools-wb-webapp/pom.xml]: The changes to the Webapp's ```pom.xml``` are *only* required if you need dynamic Bean Validation enabled. If these changes are not made to a Webapp's ```pom.xml``` the fallback position is that validation returns an empty collection of ```ConstraintViolation``` i.e. no validation is performed. I did not add this to kie-wb-distributions at the moment as there is only one consumer for dynamic bean validation; and that is Form Modeller... @pefernan these PRs may be of interest to you!

Comment (pefernan) [inline drools-wb-webapp/pom.xml]: @manstis thanks!

**Comment by csadilek** [comment]
+1

**Pull request merged**